<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>szamol</class>
 <widget class="QWidget" name="szamol">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>szamol</string>
  </property>
  <widget class="QLineEdit" name="firstNum">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>25</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="operation">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>25</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="secondNum">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>25</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>23</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="equalNum">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>26</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="generate">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Generál</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Eredmények:</string>
   </property>
  </widget>
  <widget class="QListWidget" name="results">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>151</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="opsGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>161</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Műveletek:</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>160</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="add">
       <property name="text">
        <string>összeadás</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="sub">
       <property name="text">
        <string>kivonás</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="mult">
       <property name="text">
        <string>szorzás</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="div">
       <property name="text">
        <string>osztás</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
